### DISCUSSION TITLE
I need to block projects with certain labels/topics/contributors/languages and name-substrings from recommendations

### DISCUSSION THREAD

**Discussion by jokeyrhyme** [community/community#52201] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There are some projects and technologies that I'm just flatly not interested in, and I'm tired of GitHub recommending to me in the feed and the "explore" sidebar widget

e.g. 
- it's extremely unlikely that I will develop an app for Windows or macOS, so I don't want to Swift or C# recommendations
- I don't want to see cryptocurrency or blockchain recommendations (and these are so political/philosophical that I'm surprised GitHub doesn't special-case the way these are recommended in the first place)

I want them gone

Note that I'm okay with these showing up in search results, assuming they are relevant to my search query, because that's me explicitly asking for them

This discussion is about content that I did not explicitly ask for

Cheers <3

**Comment by adlai** [comment]
I think the best approach would be to reabuse the same syntax that works for ignoring keywords when searching; although this would go on "wishlist for gamma feed", rather than "feedback on public beta"...
